protocol/indexer/off_chain_updates/off_chain_updates_test.go

```diff
@@ -363,10 +364,14 @@ func TestShouldSendOrderRemovalOnReplay(t *testing.T) {
 			orderError: errorsmod.Wrapf(clobtypes.ErrTimeExceedsGoodTilBlockTime, "wrapped error"),
 			expected:   false,
 		},
-		"Returns false for other error": {
+		"Returns true for other error": {
 			orderError: clobtypes.ErrFokOrderCouldNotBeFullyFilled,
 			expected:   true,
 		},
+		"Returns false for ErrImmediateExecutionOrderAlreadyFilled": {
```

Comment:
meganit: Can you move this test-case above the `true` test-case so that the `true` test-case is last?